templates/zerver/help/at-mention-a-team-member.md

```diff
@@ -17,6 +17,10 @@ email address or part of the username of the user you want to @-mention.
 When you begin typing an email address or username, autocomplete suggestions
 will appear.
 
+    !!! tip ""
+        You can use **@all** to mention everyone subscribed to the stream you
+        send the message to.
+
```

Comment:
Yeah, since the behavior is different enough "mention everyone" makes enough sense by itself.